funnel/models/event.py

```diff
@@ -163,6 +177,15 @@ def update_from_dict(cls, updated_participant):
         else:
             return None
```

Comment:
Silent failure is a bad idea. Also, the `db.session.add` above is a bad idea since no new participant is being created in this code block. It'll cause hard to debug problems elsewhere later.